Content.Server/Administration/Systems/BwoinkSystem.cs

```diff
@@ -327,22 +328,19 @@ private async void OnWebhookChanged(string url)
                 return;
             }
 
-            var webhookId = match.Groups[1].Value;
-            var webhookToken = match.Groups[2].Value;
-
             // Fire and forget
-            await SetWebhookData(webhookId, webhookToken);
+            await SetWebhookData(url); // Frontier - Support for Custom URLS
         }
 
-        private async Task SetWebhookData(string id, string token)
+        private async Task SetWebhookData(string url)  // Frontier - Support for Custom URLS
         {
-            var response = await _httpClient.GetAsync($"https://discord.com/api/v10/webhooks/{id}/{token}");
+            var response = await _httpClient.GetAsync(url);
```

Comment:
This is to Supply the "_webhookData" data it wants, so it add in links to the Ahelp Messages, see BwoinkSystem:385
If its not defined or empty, it will keep out that linking.

Technically seen, this do is unsafe but 1) The URL is set by the Hoster/Sudo user only and 2) If you input a non discord url, it still gets used and post requests to it, which uh... is already a secruity flaw beforehand? (could blame upstream) (before rewriting this)